test/split.jl

```diff
@@ -1,121 +1,129 @@
 using Dates
 using Test
-
 using MarketData
-
 using TimeSeries
 
-
 @testset "split" begin
 
-
-@testset "find methods" begin
```

Comment:
well, could you send the formatting patch as another PR? 
(and maybe introduce some formatter in the standalone PR, e.g. https://github.com/julia-actions/julia-format)

This cause the patch not human readable.

Reply:
#530